chef/cookbooks/provisioner/recipes/setup_base_images.rb

```diff
@@ -240,6 +240,15 @@
   package "tftp-server"
 when "suse"
   package "tftp"
+
+  # work around change in bnc#813226 which breaks
+  # read permissions for nobody and wwwrun user
+  directory tftproot do
+    recursive true
```

Comment:
does the recursive really make sense here? wouldn't this enforce ownership, group and permissions on all files/folders underneath?